rocky/katalogus/views/katalogus.py

```diff
@@ -65,27 +31,70 @@ def filter_queryset(self, queryset, filter_options):
         if filter_options == "all":
             return queryset
         if filter_options == "enabled":
-            return [plugin for plugin in queryset if plugin["enabled"]]
+            return [plugin for plugin in queryset if plugin.enabled]
         if filter_options == "disabled":
-            return [plugin for plugin in queryset if not plugin["enabled"]]
+            return [plugin for plugin in queryset if not plugin.enabled]
 
     def sort_queryset(self, queryset, sort_options):
         if sort_options == "a-z":
             return queryset
         if sort_options == "z-a":
             return queryset[::-1]
         if sort_options == "enabled-disabled":
-            return sorted(queryset, key=lambda item: not item["enabled"])
+            return sorted(queryset, key=lambda item: not item.enabled)
         if sort_options == "disabled-enabled":
-            return sorted(queryset, key=lambda item: item["enabled"])
+            return sorted(queryset, key=lambda item: item.enabled)
+
+    def sort_alphabetic_ascending(self, queryset):
+        return sorted(queryset, key=lambda item: item.name.lower())
+
+
+class KATalogusView(OrganizationView, ListView, KATalogusFilterView):
+    """View of all plugins in KAT-alogus"""
+
+    template_name = "katalogus.html"
+
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+        self.katalogus_client = get_katalogus(self.organization.code)
+
+    def get_queryset(self):
+        queryset = self.sort_alphabetic_ascending(self.katalogus_client.get_plugins())
+        return self.filter_katalogus(queryset)
```

Comment:
While this is also okay, I think it would be prettier to have a `BaseKATalogusView`  that doesn't define `template_name` and `get_queryset` and create a seperate `PluginListView` that inherits `BaseKATalogusView` similar to `BoefjeListView` and ` NormalizerListView`. I think it makes it more clear what the intent is.